### PR TITLE
Update styling for header

### PR DESCRIPTION
**Pull request by Bibek-Regmi**
## Type of Change (Select One and Apply Github Label)

- [ ] Bug fix (non-breaking change which fixes an issue) -> Apply bugfix label
- [ ] New feature (non-breaking change which adds functionality) -> Apply enhancement label
- [ ] Breaking change (fix or feature that would cause existing functionality to change) -> Apply breaking label

## Which browsers have you tested?
- [ ] Internet Explorer 11
- [ ] Edge
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

## Checklist:

- [ ] I have read the (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md] document.
- [ ] My code passes the automated linter.
- [ ] This code has been reviewed by another team member and passes the reviewer checklist found in (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md]
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My code is 508 compliant as tested by AMP and JAWS
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/packages/layouts/src/lib/feature/header/header.component.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  min-height: 3rem;
+}
+
+.sds-nav__secondary-blank {
+  bottom: 3rem;
+}
+
+.usa-navbar {
+  justify-content: flex-end;
+}
```
Comment (Bibek-Regmi) [inline libs/packages/layouts/src/lib/feature/header/header.component.scss:11]: https://github.com/GSA/sam-styles/pull/368/files

**Review by yerramshilpa — COMMENTED** [review]
Waiting for sam-style release to review the PR

File: libs/packages/layouts/src/lib/feature/header/header.component.scss (hunk 1 above)
Comment (yerramshilpa) [reply libs/packages/layouts/src/lib/feature/header/header.component.scss:11]: @Bibek-Regmi  are you still waiting on sam-styles release for it

Comment (Bibek-Regmi) [reply libs/packages/layouts/src/lib/feature/header/header.component.scss]: Yes - I got some feedback on the sam-styles PR, and updated it in sam-styles.

Comment (yerramshilpa) [reply libs/packages/layouts/src/lib/feature/header/header.component.scss]: @Bibek-Regmi  Once sam-styles done, updated the sam-styles version and verify changes

**Pull request closed without merging**